Header header
float64 delta_azimuth
float64 delta_elevation
float64 delta_distance